README.md

```diff
@@ -5,57 +5,35 @@
 
 ## How to Build
 
-### Visual Studio
+### Necessary Tools
 
-#### Necessary Tools
+* `dotnet` command line tool
 
-* Visual Studio 15
-* Nuget
-* Windows PowerShell (optional)
+### Build
 
-#### Installing Dependencies
+`dotnet build FaunaDB.Client`
```

Comment:
I think it'd be better if stated as a command. Like "Run: `dotnet build FaunaDB.Client`".